parm/config/gfs/config.atmanlvar

```diff
@@ -0,0 +1,11 @@
+#! /usr/bin/env bash
+
+########## config.atmanlvar ##########
+# Atm Var Analysis specific
+
+echo "BEGIN: config.atmanlvar"
+
+# Get task specific resources
+. "${EXPDIR}/config.resources" atmanlvar
+
```

Comment:
Did we not discuss moving the JEDI_YAML and JEDI_EXE for the variational executable here from `config.atmanl`?

Reply:
@aerorahul We did, but I reverted a bunch of changes that weren't directly related to the atmanlfv3inc job, because this PR was becoming pretty unwieldy. I can change that line back though if you want.

Reply:
lets keep this as is for now, but make an issue to discuss and hash this out sooner (rather than later).

Reply:
Will do